create.ps1

```diff
@@ -315,32 +361,42 @@ try {
     $personCount = ($correlatedPersonAccount | Measure-Object).Count
     if ($employeeCount -eq 1) {
         $actionList.Add('CorrelateAccount')
-    } elseif ($employeeCount -gt 1) {
+    }
+    elseif ($employeeCount -gt 1) {
         $actionList.Add('MultipleEmployeeFound')
-    } elseif ($personCount -eq 1) {
+    }
+    elseif ($personCount -eq 1) {
         $actionList.Add('CreateEmployee')
-        $outputContext.AccountReference = $correlatedPersonAccount.PERSONID
-    } elseif ($personCount.Count -gt 1) {
+        $outputContext.AccountReference = $correlatedPersonAccount.PERSON_PERSONID
+    }
+    elseif ($personCount -gt 1) {
         $actionList.Add('MultiplePersonFound')
-    } else {
+    }
+    else {
         $actionList.Add('CreatePerson')
         $actionList.Add('CreateEmployee')
```

Comment:
Deze query kan nu in theorie meerdere personen terug geven. Als er al een account bestond met de voor en achternaam. En je maakt er 1 extra aan. Krijg je in weze twee accounts, misschien moeten we hier ook filteren op het account zonder employee nummer. Als je dit niet doet, weet je niet welk account je zojuist hebt aangemaakt. (of je doet iets met de createddate, (mocht dat ook in het object zitten.)